tests/network-cache.js

```diff
@@ -0,0 +1,76 @@
+export default {
```

Comment:
would advise creating a subfolder or moving out of initial tests dir since its not a test, filename is a bit misleading if you dont know we use *.test.js

Reply:
Like `./test/network-cache/network-cache.js`? Or `./test/constants/network-cache.js`

Note we already have other constants declared in that directory and this is following a similar pattern.

```
./sdk> ls tests/
example-credential.js  example-schema.js  integration/  network-cache.js  poa/  test-constants.js  unit/
```

Reply:
i think we can leave it for now and clean up later